cadquery/occ_impl/exporters/assembly.py

```diff
@@ -23,34 +25,62 @@
 from OCP.Message import Message_ProgressRange
 from OCP.Interface import Interface_Static
 
-from ..assembly import AssemblyProtocol, toCAF, toVTK
+from ..assembly import AssemblyProtocol, toCAF, toVTK, toFusedCAF
 from ..geom import Location
 
 
-def exportAssembly(assy: AssemblyProtocol, path: str, **kwargs) -> bool:
+class ExportModes:
+    DEFAULT = "default"
+    FUSED = "fused"
+
+
+STEPExportModeLiterals = Literal["default", "fused"]
+
+
+def exportAssembly(
+    assy: AssemblyProtocol,
+    path: str,
+    exportMode: STEPExportModeLiterals = "default",
+    **kwargs
```

Comment:
Hm, actually why not list the prams explicitly in the signature. That is certainly more friendly for static analysis.

Reply:
Nevermind, too much scope creep